### ISSUE TITLE
Applying to okhttp result in failure

### ISSUE THREAD

**Issue by yschimke** [usefulness/maven-sympathy#24]
```
> Task :okhttp:sympathyForMrMaven FAILED
[baseline] dependency com.squareup.okhttp3:okhttp:{strictly (0,5.0.0-SNAPSHOT[} version changed (0,5.0.0-SNAPSHOT[ -> 5.0.0-alpha.14

Execution failed for task ':okhttp:sympathyForMrMaven'.
> Declared dependencies were upgraded transitively. See task output above. Please update their versions.
```

I think because okhttp has test dependencies that depend on other modules that depend on okhttp.

But also maybe an error for OkHttp to fix, I'm just not clear.

Testing on https://github.com/square/okhttp/pull/8431

**Comment by mateuszkwiecinski** [comment]
My appologies for a delayed response, I somehow didn't get (or missed) the notification. 
So, what my quick research established:
- the transitively updated depdendency comes from [`biz.aQute.bnd.builder`](https://github.com/bndtools/bnd/blame/bbf85f4ae1c451e60a50ff65dd11d901c2f31664/gradle-plugins/biz.aQute.bnd.gradle/src/main/java/aQute/bnd/gradle/BndBuilderPlugin.java#L113) plugin, which creates the `baseline` configuration and sets dynamic(?) version which is later resolved to a specific one. The failure of the plugin was an _expected_ behavior, but not necessairly _valid_ since the configuration wasn't published via Maven. 
- I just released [`0.3.0`](https://github.com/usefulness/maven-sympathy/releases/tag/v0.3.0) which changes default behavior and should only check configurations that are actually published, effectively ignoring the `baseline` configuration (but if it was for some reason included in maven-publish publishing it would start failing the build again)
- The new implementation, applied to the okhttp the same way it's done in the linked PR, will run against `compileClasspath` and `runtimeClasspath` configurations only (which according to my understanding are the only ones taken into account when publishing the artifacts)

**Comment by yschimke** [comment]
Thank you, that's great.
